sql/table.cc

```diff
@@ -8751,6 +8751,7 @@ bool TR_table::update(ulonglong start_id, ulonglong end_id)
     return true;
 
   store(FLD_BEGIN_TS, thd->transaction_time());
+  thd->set_time();
```

Comment:
I assume that you have now done proper regression testing running other relevant suites including --suite=wsrep,galera using galera library ?

Reply:
We'll see it here soon in [buildbot](http://buildbot.askmonty.org/buildbot/grid?category=main&branch=tt-10.1-MDEV-16211).

This particular code executed only with trx_id-based system versioning. And there was not Galera tests for that case. So nothing should be broken by this change.